test-clients/src/main/java/com/hedera/services/bdd/suites/perf/SubmitMessageLoadTest.java

```diff
@@ -195,7 +197,8 @@ private static HapiApiSpec runSubmitMessages() {
 						TOPIC_EXPIRED,
 						INVALID_TOPIC_ID,
 						INSUFFICIENT_PAYER_BALANCE)
-				.hasKnownStatusFrom(SUCCESS, OK, INVALID_TOPIC_ID)
+				.hasKnownStatusFrom(SUCCESS, OK, INVALID_TOPIC_ID, INSUFFICIENT_PAYER_BALANCE
+						,UNKNOWN,TRANSACTION_EXPIRED)
```

Comment:
Why is UNKNOWN added as Known status?

Reply:
This part is tricky. I am not sure if some recent changes is causing this or not. In several test runs, the client logs has marked the `UNKNOWN` as reason for suite failure by the test suite. And the summary of the suite FAILED is picked up by client validator (we can also let the HAPIClientValidator to ignore this message, which I am not sure is a good idea or not).  IMO, at end of the transaction duration, there should be a clear decision for this transaction. The `UNKNOWN` status shouldn't appear as the response to the test client at all. I think until we have a larger scope discussion on the error handling, we won't have a clear answer for this. 

Or if anyone has a better suggestion, I will be glad to follow.